### PR TITLE
Two order colleen

### PR DESCRIPTION
**Pull request by ColleenWoolsey**
Fixes - Added PaymentType, OrderProduct and ProductType models

 Bug fix (non-breaking change which fixes an issue)
 New feature (non-breaking change which adds functionality)
 
Testing Instructions
1. git fetch --all
2. git checkout two-order-colleen
3. checkout all models for attributes and code

Checklist:
 My code follows the style guidelines of this project
 I have performed a self-review of my own code
 I have added test instructions that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by Daniel-Brewer — APPROVED** [review]
Looks good

**Review by zwcrawford — APPROVED** [review]
Approved. Models look good.

File: BangazonAPI/Models/Order.cs [hunk 1]
```diff
@@ -1,11 +1,25 @@
...
+        public int CustomerId { get; set; }
+
+        [Required]
+        public int PaymentTypeId { get; set; }
+
+        public Customer Customer { get; set; }
+
+        public PaymentType PaymentType { get; set; }
+
+        // public List<Exercise> Exercises { get; set; } = new List<Exercise>();
```
Comment (askingalot) [inline BangazonAPI/Models/Order.cs:23]: Remove commented code

File: BangazonAPI/Controllers/OrderController.cs [hunk 2]
```diff
@@ -0,0 +1,210 @@
+﻿using System;
```
Comment (askingalot) [inline BangazonAPI/Controllers/OrderController.cs:1]: I thought this PR was just for models?

**Review by askingalot — CHANGES_REQUESTED** [review]

**Pull request closed without merging**